src/Chainweb/Pact/TransactionExec.hs

```diff
@@ -881,6 +894,17 @@ disablePact40Natives ec = if has (ecFlags . ix FlagDisablePact40) ec
       , "str-to-list"]
 {-# INLINE disablePact40Natives #-}
 
+-- | Disable certain natives around pact 4.2.0
+--
+disablePact420Natives :: ExecutionConfig -> EvalEnv e -> EvalEnv e
```

Comment:
Add a TODO comment to provide a better solution than this copypasta of the 40 code above.